src/utils/dates.js

```diff
@@ -10,3 +12,101 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export const handleNextPurchaseDate = (
+	name,
+	currentDate,
+	dateCreated,
+	purchasesCount,
+	dateNextPurchased,
+	dateLastPurchased,
```

Comment:
Typically when you have so many properties being passed into a util method you've defined, it's helpful to define the inputs as an object with these named properties. That way adding in the values in the wrong order is less likely.
```suggestion
	{
	name,
	currentDate,
	dateCreated,
	purchasesCount,
	dateNextPurchased,
	dateLastPurchased,
	}
```

Reply:
makes so much sense! fixed!